app/enterprise/1.3-x/plugin-development/custom-logic.md

```diff
@@ -263,34 +263,54 @@ Plugin                      | Priority
 ----------------------------|----------
 pre-function                | `+inf`
 zipkin                      | 100000
-ip-restriction              | 3000
```

Comment:
It was wrong for Kong EE.  See: https://github.com/Kong/kong-ee/blob/master/kong/plugins/ip-restriction/handler.lua#L14  (Only Konger can see this link 😉 )

Reply:
@nijikokun for your 👀 since you bumped up the priority of ip-restriction for CE [here](https://github.com/Kong/kong/pull/3516).  *this is EE.